tests/test_generate_mbz_toc.py

```diff
@@ -198,10 +200,10 @@ def test_toc_creation_multiple_sections(
     html_path.mkdir()
     mbz_builder(mbz_path, activities=[page_1, page_2], sections=sections)
     mbtools.extract_html_content.replace_content_tags(mbz_path, html_path)
-    md_filepath = tmp_path / "toc.md"
```

Comment:
You can probably revert this change you made (which would cascade down to various other small, unnecessary changes in this PR).